TradeItIosTicketSDK2/EquityPreviewCellFactory.swift

```diff
@@ -72,6 +72,19 @@ class EquityPreviewCellFactory: PreviewCellFactory {
 
     // MARK: Private
 
+    private func formatQuantity(rawQuantityType: String, quantity: NSNumber) -> String{
```

Comment:
Ticket: https://www.pivotaltracker.com/story/show/157754041

Not very happy with the method but this is my first stab at it. Notes: 

- The `NumberFormatter.formatQuantity` defaults to 2, we could change/add "Quantity formatter" method that uses `orderCapabilities?.maxDecimalPlacesFor`.
- `orderDetails.orderQuantityType` is a string that in order to use it with `orderCapabilities?.maxDecimalPlacesFor` need to be convert to enum. If `EquityPreviewCellFactory.swift`  received the full `order` object or orderQuantityType enum the conversion will not be needed.

Reply:
@falnatsheh yeah I think adding a new parameter to `NumberFormatter.formatQuantity` makes sense